src/atoms/CustomHtml/CustomHtml.test.js

```diff
@@ -162,10 +69,8 @@ test('If no fallback src is provided, and there is an unsourced script, place it
     />
   );
 
-  expect(container.firstChild.innerHTML).toEqual(
-    '<iframe width="100%" height="500px" frameborder="0" scrolling="yes" marginheight="0" marginwidth="0" src=""><div class="typeform-widget" data-url="https://mprnews.typeform.com/to/y5uiHF" style="width: 100%; height: 500px;"></div> <script> (function() { var qs,js,q,s,d=document, gi=d.getElementById, ce=d.createElement, gt=d.getElementsByTagName, id="typef_orm", b="https://embed.typeform.com/"; if(!gi.call(d,id)) { js=ce.call(d,"script"); js.id=id; js.src=b+"embed.js"; q=gt.call(d,"script")[0]; q.parentNode.insertBefore(js,q) } })() </script> <div style="font-family: Sans-Serif;font-size: 12px;color: #999;opacity: 0.5; padding-top: 5px;"> powered by <a href="https://admin.typeform.com/signup?utm_campaign=y5uiHF&amp;utm_source=typeform.com-13901520-ProPlus3&amp;utm_medium=typeform&amp;utm_content=typeform-embedded-poweredbytypeform&amp;utm_term=EN" style="color: #999" target="_blank">Typeform</a> </div></iframe>'
-  );
-  expect(container.querySelectorAll('iframe').length).toEqual(1);
+  expect(container.firstChild.innerHTML).toMatch(/<\/script>$/);
+  expect(container.querySelectorAll('script').length).toEqual(1);
 });
 
 test('If the fallback src is not on our whitelist, and the script has no source, all html is placed in iframe', () => {
```

Comment:
Is this still true?

Reply:
I need to update the test descriptions - doing now